demo/addons/fmod/tool/property_editors/FmodEventEditorProperty.gd

```diff
@@ -0,0 +1,84 @@
+@tool class_name FmodEventEditorProperty extends FmodPathEditorProperty
+
+
+static var EVENT_PARAMETER_PREFIX_FOR_PROPERTIES = "event_parameter"
```

Comment:
caveat - using static var makes this addon incompatible with Godot 4.0 but only supports Godot 4.1 (we may want to reflect this in our compatibility matrix)

Reply:
We do not use compatibility matrix anymore.  
We now release using this kind of tags: `x.y.z-X.Y.Z`, where `X.Y.Z` is godot version.

Reply:
If I remember Godot 4.1 extensions are not compatible with Godot 4.0 anyway, As Godot refactor their API in 4.1 in favor of a new "forward compitable" model.